x11-terms/alacritty/alacritty-9999.ebuild

```
@@ -57,7 +57,7 @@ RDEPEND="${COMMON_DEPEND}
 BDEPEND="
 	dev-util/cmake
 	>=virtual/rust-1.57.0
```

```suggestion
	>=virtual/rust-1.65.0
```

That should reflect what upstream has.